src/hotspot/share/nmt/mallocTracker.cpp

```diff
@@ -43,7 +43,7 @@
 #include "utilities/ostream.hpp"
 #include "utilities/vmError.hpp"
 
-size_t MallocMemorySummary::_snapshot[CALC_OBJ_SIZE_IN_TYPE(MallocMemorySnapshot, size_t)];
+MallocMemorySnapshot MallocMemorySummary::_snapshot{};
```

Comment:
Why the curly braces?

Reply:
They're not necessary, I just have a tendency to explicitly use brace/"uniform" initialization everywhere. I've removed them.